### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by tirthajyoti-ghosh**
# Weather App

This is a weather app that shows temperatures (current, feels like, min, max), cloudiness (%), and wind speed (m/s) by city name. The city name is obtained from form input. This app consumes OpenWeather API in the `JSON` format to show weather data.

Data points consumed:

* City name
* Country code
* Weather description
* Weather icon
* Current temperature
* Feels like temperature
* Minimum temperature
* Maximum temperature
* Cloudiness
* Wind speed

## Usage

Usage example

1. Enter the city name and press the `Enter` key.
2. Weather data is displayed here.
3. These are buttons to convert the temperature in their respective units.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    cityName: response.name,
+    countryCode: response.sys.country,
+    description: response.weather[0].description,
+    icon: response.weather[0].icon,
+    temperature: response.main.temp,
+    feelsTemp: response.main.feels_like,
+    minTemp: response.main.temp_min,
+    maxTemp: response.main.temp_max,
+    clouds: response.clouds.all,
+    wind: response.wind.speed,
```
Comment (suretrust) [inline src/main.js]: * [x] ES6 - use object destructuring to get the values from an object. This way you can avoid repeating a lot of code. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/destructuring_requirement.png) from Pathwright)
```javascript
const { name, sys, weather, main, clouds, wind } = response
```

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team, 👋 

### Great job thus far! 🙌 Please see my comments. 👇 

* [x] Your app's appearance should change based on the received data. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))

* [x] Keep the application logic separated from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright)

**KIndly submit another CRR once these issues have been solved.** _Happy coding!_ 🎉

**Comment by tirthajyoti-ghosh** [comment]
Thanks to @suretrust for the review.

Changes made after review:

- Add destructuring 
  - Separate DOM from logic
  - Remove wet code

Regarding this - 
>Your app's appearance should change based on the received data.

The weather image on the right side changes based on the weather. 

image

image

image

As you can see, the image on the right changes.

I have opted for a simple and clean UI. So I have not added any colors or GIFs.

Looking forward to the next review.

**Review by whiz25 — APPROVED** [review]
## Approved :clap: 
You have done the minimum required for this PR. Please merge and proceed to the next project.

### Martin Afani

**Pull request merged**